sensorlib/ReadMe.md

```diff
@@ -12,6 +12,14 @@ Raw sensor observations from the CARLA sensor are composed to build a dataset re
 
 This new feature is provided to enable vulnerable road user detection using a LIDAR sensor in the context of a CARLA-driven simulation. The data is intended to feed into a CARMA V2X system for data fusion with other sensors and action by CARMA platform vehicles.
 
+## Interface
+
+The XML-RPC interface consists of one function to retrieve the list of detected objects.
```

Comment:
There should be 2 functions:
- One to create a sensor at a given location/orientation.
- One to retrieve all detected objects from the sensor.

Reply:
Are we only exposing as a server? If so then yes. I was under the impression some other Python code would configure the sensor and then we serve the detected objects, but if this needs to be a standalone server you are correct.